website/src/pages/download/index.tsx

```diff
@@ -23,7 +23,7 @@ const MAC_ARM = 'macOS (ARM, 64-bit)';
 const Releases: FC = (): ReactElement=> {
   const [cacheTagName] = useLocalStorageState<string>('global-cache-tag-name');
   const tagName = cacheTagName as string || 'v0.8.25';
-  const DOWN_LINK = 'https://github.com/datafuselabs/databend/releases/download/';
+  const DOWN_LINK = 'https://repo.databend.rs/databend/';
```

Comment:
```suggestion
  const DOWNLOAD_LINK = 'https://repo.databend.rs/databend/';
```

I prefer make reader happy (